ncm-symlink/src/main/perl/symlink.pod

```diff
@@ -28,92 +25,148 @@ Target definition and contextual variable value may also contain expression inte
 
 A list of symbolic links to create or delete.  Each entry
 must be of the structure_symlink_entry type which has the following
-fields: name, target, delete, exists, replace.
+fields: 
 
-'name' is the symbolic link name, 'target' is the link target, other fields are boolean. 'delete' means delete the link rather than create. 'exists' means check the target exists when creating or check the link exists when deleting. 
+=over
 
-'replace' is an option taking one or several of the several following suboptions : all, dir, dirempty, file, link, none. Each suboption can have the following values : 'yes', 'no', a string interpreted as a file extension used to rename the existing object with the link name. If a suboption value is empty, it is interpreted as 'yes'. If value is 'yes', replacement is enabled but existing object is not renamed, except if a default extension has been defined with 'none' or 'all' or if it is a non empty directory (renamed with a default extension).
+=item *
 
-When 'all' or 'none' suboption is specified, other suboptions can be specified too to redefine extension or to prevent replacement of a specific object type. 'dirempty' means directory is replaced only if empty. 'none=extension' can be used to establish a default backup extension without enabling replacement (as it is with 'all').
+C<name>: symbolic link name (path).
 
-=head2 /software/components/symlink/context
+=item *
 
-A list of contextual variables to use in target definitions.  Each entry
-must be of the structure_symlink__context_entry type which has the following
-fields: name, value.  Contextual variables are global.
+C<target>: link target path. The target path can be built using a command output with the command to execute between a pair of C<@@> or a contextual variable using the syntax
+C<{variable}> (variables are defined in C</software/components/symlinks/context>). Unless the shell command between C<@@> must be reevaluated for each link, it is better 
+to associate the shell command
+with a contextual variable and use the variable in the target definition, as a contextual variable is evaluated once (global). 
```

Comment:
add example `C<"/software/components/symlinks/context/myvariable" = "@mycommand --option@";>` (unless i'm mistaken with the inner details)